src/ds/d400/d400-device.cpp

```diff
@@ -1081,7 +1101,28 @@ namespace librealsense
         //TODO: Implement
     }
 
-    platform::usb_spec d400_device::get_usb_spec() const
+    // Check if need change camera name due to number modifications on one device PID.
+    void update_device_name(std::string& device_name, const ds::d400_caps cap, const uint8_t cur_gvd_version)
+    {
+		switch (cap)
+		{
+		    case ds::d400_caps::CAP_IR_FILTER:
+			    if (is_capability_supports(cap, cur_gvd_version))
+                    device_name += "F"; // Adding "F" to end of device name if it has IR filter.
+                break;
+
+		    case ds::d400_caps::CAP_IP65:
+                if (is_capability_supports(cap, cur_gvd_version))
+                {
+                    // device_name[device_name.size() - 1] = '6'; // Changing last digit name to "6" if device has IP65 standart.
+                    std::regex vowel_re("D455");
+                    std::regex_replace(device_name, device_name.begin(), device_name.end(), vowel_re, "D456");
+                }
+                break;
```

Comment:
Please add a default case (always)
Here you can throw and say the required capability is not supported for name update (with adding the cap value)

Reply:
The default case was added.
I implemented throw in the default case but can't find it in IDE after throwing an exception.
As a result, the viewer continues work after throwing an exception and NOT showing the device name.
Please advise, if we can leave it as is or improve it.